### DISCUSSION TITLE
Feature Request: Range Field

### DISCUSSION THREAD

**Discussion by thepedroferrari** [keystonejs/keystone#5308] [Feature requests]
## Range field
I've been implementing myself on the frontend and adding the required constraints there before I send the update request, but I believe it would be awesome if we could have a Range field in Keystone.

Very similar to the integer field with an extra set of rules. Every time I am looking at a scale, I wish there were more fine-grain control over the min and max values.

### Proposed settings:

- Default Value: 0
- Minimum Value: The minimum (numeric) value allowed. Defaults to 0.
- Maximum Value: The maximum (numeric) value allowed. Defaults to 100.
- Step Size: The increment at which a numeric value can be set. Defaults to 1.
- isRequired: ...
- isIndexded: ...
- isUnique: ...

### Use cases:
- Rating
- Temperatures
- Rotation of a circle
- Audio controls
- Color codes
- Weight
and ofc many others.

### UI:
Something that could make use of `<input type="range">`

### Alternative:
Adding min and max values to the integer field could be just enough.

I would love to help with a PR if you believe this to be a good idea.

**Comment by bladey** [comment]
Hey @thepedroferrari, thank you for this detailed field suggestion, it looks great.

We aren't looking for new fields right now but once we cross into our GA release we'll be looking for community contributions.
